src/components/molecules/Visualizer/Engine/Cesium/common.ts

```diff
@@ -633,6 +634,32 @@ export async function flyToGround(
   );
 }
 
+export function getPixelDistance(viewer: Viewer, sample = "bottom") {
+  const scene = viewer.scene;
+  const camera = scene?.camera;
```

Comment:
Currently you aren't using this camera variable (you probably aren't getting a warning because you do the if !camera check).
You are also using canvas and globe from the scene below, so you can clean up by doing

`const { camera, canvas, globe } = scene;`

Whether you do this suggestion or not, please use camera where you are declaring `left` and `right` ;)

Reply:
Updated. I found some check or definition is unnessary and removed them.